### PR TITLE
MS regular exemptions

### LINKED ISSUE #2190: MS regular exemptions
MS regular exemptions Line 1-5

$12,000 married (filing jointly or combined return, widow: spouse died) (Line 1- 2)
$6,000 married (filing separately, each get $6,000) (Line 3)
$8,000 Head of Family (Line 4)
$6,000 Single (Line 5)

https://www.dor.ms.gov/sites/default/files/Forms/Individual/80100221.pdf#page=5

Miss. Code Ann. § 27-7-21 (b-d)
https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=adb5f365-0449-4d49-80f0-22673c0d0165&nodeid=AAPAAFAABAAP&nodepath=%2fROOT%2fAAP%2fAAPAAF%2fAAPAAFAAB%2fAAPAAFAABAAP&level=4&haschildren=&populated=false&title=%C2%A7+27-7-21.+Exemptions+allowed.&config=00JABhZDIzMTViZS04NjcxLTQ1MDItOTllOS03MDg0ZTQxYzU4ZTQKAFBvZENhdGFsb2f8inKxYiqNVSihJeNKRlUp&pddocfullpath=%2fshared%2fdocument%2fstatutes-legislation%2furn%3acontentItem%3a8V3T-78H2-D6RV-H4WK-00008-00&ecomp=8gf5kkk&prid=e1d2bafd-2018-44e8-a4fc-e9e2b08a4daa

### PR DESCRIPTION
**Pull request by ZiqianCui**
Fixes #2190

MS regular exemptions

### CODE REVIEW

File: policyengine_us/parameters/gov/states/ms/tax/income/exemptions/regular/amount.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+description: MS provides a regular exemption of $6k for individuals, $12k for couples, $8k for heads of household.
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/ms/tax/income/exemptions/regular/amount.yaml:1]: Spell out states and avoid including values in descriptions.
```suggestion
description: Mississippi provides an exemption of this value depending on filing status.
```

File: policyengine_us/parameters/gov/states/ms/tax/income/exemptions/regular/amount.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+metadata:
+  unit: currency-USD
+  label: MS Regular Exemption
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/ms/tax/income/exemptions/regular/amount.yaml:4]:
```suggestion
  label: Mississippi exemption
```

File: policyengine_us/parameters/gov/states/ms/tax/income/exemptions/regular/amount.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  reference: 
+    - title: Mississippi State FORMS 80-105 AND 80-205
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/ms/tax/income/exemptions/regular/amount.yaml:6]: Can you find a legal reference?

Comment (ZiqianCui) [reply policyengine_us/parameters/gov/states/ms/tax/income/exemptions/regular/amount.yaml:6]: Sure!

Comment (MaxGhenis) [reply policyengine_us/parameters/gov/states/ms/tax/income/exemptions/regular/amount.yaml:6]: I mean from https://advance.lexis.com/container?config=00JAAzNzhjOTYxNC0wZjRkLTQzNzAtYjJlYS1jNjExZWYxZGFhMGYKAFBvZENhdGFsb2cMlW40w5iIH7toHnTBIEP0&crid=2fcdae0c-c595-411f-aa8c-6818383d928e

**Review by MaxGhenis — COMMENTED** [review]
could you run `make format`?

**Comment by MaxGhenis** [comment]
Looks good - just needs a `make format` (ensure you have linecheck installed)

**Pull request merged**